src/lib/core/CHIPConfig.h

```diff
@@ -2400,6 +2400,16 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_COMMISSIONING_HINT_INDEX_PRESS_RESET_UNTIL_BLINK_WITH_POWER 11
 #endif
 
+/**
+ * @def CHIP_CONFIG_MDNS_MAX_RESOLVER_DELEGATES
+ *
+ * @brief
+ *      Define the maximum number of delegates that will be informed of node address resolution.
+ */
+#ifndef CHIP_CONFIG_MDNS_MAX_RESOLVER_DELEGATES
+#define CHIP_CONFIG_MDNS_MAX_RESOLVER_DELEGATES 4
+#endif
```

Comment:
It's not clear that this is shared across all controllers, and that this caps the max number of controller objects you can use

Reply:
@vivien-apple Is this platform overridable? If so, I think this can be resolved, given this is one of many configs that can be overridden per platform.

Reply:
> 

Yes, the platform can override it.